lib/sequence/binlog.js

```diff
@@ -29,6 +29,7 @@ module.exports = function(options) {
   };
 
   Binlog.prototype['OkPacket'] = function(packet) {
+    // TODO: Remove console statements to make module ready for widespread use
     console.log('Received one OkPacket ...');
```

Comment:
Hmm.. if I turn the `debug` setting for the test suite to true, it outputs all a few `OkPacket`s but I never see this console message